akka-docs/src/main/paradox/stream/operators/Source-or-Flow/scanAsync.md

```diff
@@ -30,3 +30,10 @@ Note that the `zero` value must be immutable.
 
 @@@
 
+## Examples
```

Comment:
A short description what the sample does would be good as well. Also add a caveat about "In an actual application the future would probably involve some external API that returns a @scala[Future] @java[CompletionStage] rather than an immediately completed value"